src/views/Home/index.view.tsx

```diff
@@ -3,6 +3,7 @@ import HeroView from "views/Hero/index.view";
 import Button from "components/Button/index.view";
 import EmailSubscriptionForm from "components/EmailSubscription/index.view";
 import MLHBanner from "components/MLHBanner/index.view";
+import MilestonesComponent from "./components/Milestones/index.view";
```

Comment:
No need to include `./` in here as part of your path since all paths are relative to `src` 

```suggestion
import MilestonesComponent from "components/Milestones/index.view";
```

Reply:
I'm keeping this because this is in the `components/` directory inside `views/Home`. 

Otherwise it'd be `import MilestonesComponent from "views/Home/components/Milestones/index.view";`

Reply:
No you're totally right here! I just realized you're referencing the components within the `Home` view. This is my bad!